build.gradle

```diff
@@ -35,6 +35,15 @@ dependencies {
     compile group: 'com.lesfurets', name:'jenkins-pipeline-unit', version: '1.13'
     compile group: 'com.cloudbees', name: 'groovy-cps', version: '1.31'
     compile group: 'org.yaml', name: 'snakeyaml', version: '1.29'
+    constraints {
+        implementation("org.apache.logging.log4j:log4j-core") {
+            version {
+                strictly("[2.16, 3[")
```

Comment:
The bracket needs to be facing inward `[2.16, 3]`

Reply:
This is from the gradle folks https://blog.gradle.org/log4j-vulnerability

Reply:
Interesting!!